recipe/meta.yaml

```diff
@@ -3,45 +3,381 @@
 {% set sha256 = "627ec53fab43d06c1b5c950e217fa9819e169daf753111a7f244e94bf8fb3384" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-feedstock
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   noarch: python
-  number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 1
 
-requirements:
-  build:
-    - pip
-    - python
-    - setuptools
+outputs:
+  - name: {{ name }}
+    build:
+      noarch: python
+      script: python -m pip install . --no-deps --ignore-installed --verbose
+    requirements:
+      host:
+        - python
+        - pip >=18.1
+        - protobuf >=3.0.0
+        - setuptools >=34.0.0
+      run:
+        - {{ pin_compatible('protobuf', min_pin='x', max_pin='x') }}
+    test:
+      imports:
+        - google.api
+        - google.api.annotations_pb2
+        - google.api.auth_pb2
+        - google.api.backend_pb2
+        - google.api.billing_pb2
+        - google.api.client_pb2
+        - google.api.config_change_pb2
+        - google.api.consumer_pb2
+        - google.api.context_pb2
+        - google.api.control_pb2
+        - google.api.distribution_pb2
+        - google.api.documentation_pb2
+        - google.api.endpoint_pb2
+        - google.api.field_behavior_pb2
+        - google.api.http_pb2
+        - google.api.httpbody_pb2
+        - google.api.label_pb2
+        - google.api.launch_stage_pb2
+        - google.api.log_pb2
+        - google.api.logging_pb2
+        - google.api.metric_pb2
+        - google.api.monitored_resource_pb2
+        - google.api.monitoring_pb2
+        - google.api.quota_pb2
+        - google.api.resource_pb2
+        - google.api.service_pb2
+        - google.api.source_info_pb2
+        - google.api.system_parameter_pb2
+        - google.api.usage_pb2
+        - google.longrunning
+        - google.longrunning.operations_pb2
+        - google.longrunning.operations_proto
+        - google.longrunning.operations_proto_pb2
+        - google.protobuf
+        - google.protobuf.any_pb2
+        - google.protobuf.any_test_pb2
+        - google.protobuf.api_pb2
+        - google.protobuf.compiler
+        - google.protobuf.compiler.plugin_pb2
+        - google.protobuf.descriptor
+        - google.protobuf.descriptor_database
+        - google.protobuf.descriptor_pb2
+        - google.protobuf.descriptor_pool
+        - google.protobuf.duration_pb2
+        - google.protobuf.empty_pb2
+        - google.protobuf.field_mask_pb2
+        - google.protobuf.internal
```

Comment:
Do we want to be adding all these internal items to the tests?

Reply:
All the packages listed in test imports are imported by googleapis-common-protos or googleapis-common-protos-grpc.

It helps verify all the googleapis-common-protos dependencies are installed and available.